developer-docs-site/docs/guides/system-integrators.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "System Integrators Guide"
+slug: "system-integrators"
+---
+
+# System Integrators Guide
+
+This guide will walk you through all you need to successfully integrate with the Aptos blockchain. This guide assumes familiarity with blockchains, command-line interfaces, and other concepts discussed throughout the developer site. Where possible links will be made to relevant topics.
+
+An integrator may follow some or all of these steps:
+* Create an account on the blockchain
+* Exchange account identifiers with another entity on the blockchain to perform swaps, for example
+* Create a transaction
+* Obtain a gas estimate and validate the transaction for correctness
+* Ask the user to verify the intent and cost of the transaction
```

Comment:
how is this done? maybe more content on this bullet?

Reply:
Part of these are just bullets to call out what is covered below.